docs/odh/adding_kfdefs.md

```diff
@@ -0,0 +1,16 @@
+## Instructions on how to add your KFDEF
+
+Due to security reasons, we do not allow users to deploy their own `Kfdefs` (see [here][1] for details). However, should users want to request a `Kfdef` to their namespace, they can follow the instructions below for submitting a PR.
+
+Steps:
+Let `ENV`, `CLUSTER`, and `NAMESPACE` be the environment, cluster, and namespace you would like to deploy this `Kfdef` to respectively.
+1. Add your `Kfdef` to `operate-first/apps/kfdefs/overlays/$ENV/$CLUSTER/$NAMESPACE/kfdef.yaml`, accompanied by a `Kustomization.yaml` that includes the `kfdef.yaml`.
```

Comment:
I'd drop the:
```suggestion
1. Add your `Kfdef` to `kfdefs/overlays/$ENV/$CLUSTER/$NAMESPACE/kfdef.yaml`, accompanied by a `Kustomization.yaml` that includes the `kfdef.yaml`.
```

since we're already in this repo.

Reply:
If we put this on the site then it may not be clear what repo we're in. What about adding a "clone this repo" set above instead?

Reply:
Updated, let me know if this works.